.devcontainer/templates/node/.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
```

Comment:
why is this file in templates/node? isn't this common to all stacks?

Reply:
`templates/node` is the only template available right now, it is composed by just the common base (`az, gh, pre-commit, terraform` and the `node` binary with corepack)